engine/runtime/src/main/java/org/enso/interpreter/instrument/ReplDebuggerInstrument.java

```diff
@@ -155,6 +163,26 @@ public Either<Exception, Object> evaluate(String expression) {
       }
     }
 
+    /**
+     * Returns the String representation of the provided object as defined by Enso {@code to_text}
+     * operation.
+     */
+    public Either<Exception, String> showObject(Object o) {
+      try {
+        InteropLibrary atoms = InteropLibrary.getUncached();
+        Text text = TypesGen.expectText(atoms.invokeMember(o, "to_text"));
+        return new Right<>(toJavaStringNode.execute(text));
+      } catch (UnsupportedMessageException
+          | ArityException
+          | UnknownIdentifierException
+          | UnsupportedTypeException
+          | UnexpectedResultException e) {
```

Comment:
Is this list exhaustive, or can it be extended in the future? Would it be better to check for more general `InteropException` instead?

Reply:
It's taken from `Atom::toDisplayString`. I think it detects situations like when the `to_text` method could not be called due to wrong arity, return type etc. I think catching any `InteropException` will be to wide.